node-db/week-16/lesson.md

```diff
@@ -0,0 +1,89 @@
+# Node 1
+
+**What we will learn today?**
+
+* Why we need databases.
+* Creating a database with SQL, storing data in it.
+* Retrieving data from a database using SQL.
+
+THE PROBLEM
+-----------
+
+* Take existing code that stores reservation as a JSON snippet with subkeys for customer and rooms.
+
+* Pretend to be an idiot junior programmer. Idiot junior programmer swaps out the code that puts a room in a reservation object and now it just puts null.
```

Comment:
Hoping `idiot` won't make it to the final version :)

Reply:
No, this was more of a brain dump :) We need to separate it out into syllabus content and teaching notes.